src/cli_plugin/install/download.js

```diff
@@ -8,8 +9,10 @@ export function _downloadSingle(settings, logger, sourceUrl) {
 
   if (/^file/.test(urlInfo.protocol)) {
     downloadPromise = downloadLocalFile(logger, decodeURI(urlInfo.path), settings.tempArchiveFile);
-  } else {
+  } else if (/^https?/.test(urlInfo.protocol)) {
```

Comment:
I think we should check for the `//` as well, otherwise you couldn't name plugins anything that begins with "https".  We probably won't ever have a pack named that, but if we did, it would be a really weird bug to track down.